contracts/QuestFactory.sol

```diff
@@ -329,6 +329,14 @@ contract QuestFactory is Initializable, LegacyStorage, OwnableRoles, IQuestFacto
         );
     }
 
+    function cancelQuest(string calldata questId_) external {
+        Quest storage _questData = quests[questId_];
+        if (_questData.questCreator != msg.sender) revert Unauthorized();
+        IQuestOwnable quest = IQuestOwnable(_questData.questAddress);
+        quest.cancel();
+        emit QuestCancelled(_questData.questAddress, questId_, quest.endTime());
```

Comment:
@ccashwell Can you double check? I think this event is emitting the old `endTime`

Reply:
Some context: https://sepolia.etherscan.io/tx/0x0c9044b4deef12007d85ea74059a642da4e7a15c2c8f1a1fc8dada820b2cbb82#eventlog

The value of `1716526800` is not 15 mins from block time (at time of this comment)

Reply:
Actually I think I know why this is! This Quest contract was created before the latest changes

On a new Quest deployment, the logic is working as expected 👍